File: Vid1112/app/src/main/java/com/example/vid1112/MainActivity.java
package com.example.vid1112;

import android.content.Intent;
import android.example.firebase.R;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    private Button btCreateDB;
    private Button btViewDB;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        btCreateDB = (Button) findViewById(R.id.bt_createdata);
        btViewDB = (Button) findViewById(R.id.bt_viewdata);

        btCreateDB.setOnClickListener(new View.OnClickListener() {
            public void onClick(View view) {
                startActivity(FirebaseDBCreate.getActIntent(MainActivity.this));
            }
        });

        btViewDB.setOnClickListener(new View.OnClickListener() {
            public void onClick(View view) {
                Intent i = new Intent(MainActivity.this, GetFirebaseActivity.class);
                startActivity(i);
            }
        });
    }
    public void onBackPressed() {
        finishAffinity();
    }
}

File: UAS/UASPPB_45UG_11667/app/src/main/java/com/example/uasppb_45ug_11667/MainActivity.kt
package com.example.uasppb_45ug_11667

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.uasppb_45ug_11667.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    //        BINDING
    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        //BINDING
        binding = ActivityMainBinding.inflate(layoutInflater)
        val view = binding.root
        setContentView(view)

        //JANGAN LUPA BINDING
        binding.loginAdminBtn.setOnClickListener {
            startActivity(Intent(this, LoginPage::class.java))
        }

        binding.bukaMenuBtn.setOnClickListener {
            startActivity(Intent(this, DaftarMenu::class.java))
        }


    }
}